### PR TITLE
runtime/themes: Add "naysayer" theme

### PR DESCRIPTION
**Pull request by nesv**
This theme is a port of https://github.com/nickav/naysayer-theme.el.

### CODE REVIEW

File: runtime/themes/naysayer.toml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+"ui.background" = { bg = "bg" }
+"ui.text" = "text"
+"ui.linenr" = { bg = "bg", fg = "line-fg" }
+"ui.linenr.selected" = { bg = "highlight-line", fg = "line-fg" }
+"ui.selection" = { bg = "selection" }
+"ui.cursorline" = { bg = "highlight-line" }
+"ui.statusline" = { fg = "bg", bg = "text" }
+"ui.statusline.inactive" = { fg = "text", bg = "bg" }
+"ui.virtual" = "indent"
```
Comment (the-mikedavis) [inline runtime/themes/naysayer.toml:14]: Looks like this is not defined in the palette

File: runtime/themes/naysayer.toml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+"ui.virtual.ruler" = { bg = "lign-fg" }
```
Comment (the-mikedavis) [inline runtime/themes/naysayer.toml:15]:
```suggestion
"ui.virtual.ruler" = { bg = "line-fg" }
```

Comment (nesv) [reply runtime/themes/naysayer.toml]: Thank you for catching that typo!

File: runtime/themes/naysayer.toml (hunk 1 above)
Comment (nesv) [reply runtime/themes/naysayer.toml:14]: :facepalm:

**Comment by nesv** [comment]
@the-mikedavis Thank you very much for taking the time to review this!

I've made the requested fixes. If all looks good, I can rebase out the fixup commit prior to merging.

**Comment by nesv** [comment]
Rebased onto the latest `master`, to squash out the fixup commits.

**Pull request merged**